eclass/libretro-core.eclass

```diff
@@ -119,8 +119,8 @@ libretro-core_src_compile() {
 	use custom-cflags || filter-flags -O*
 	emake CC="$(tc-getCC)" CXX="$(tc-getCXX)" \
 		"$(usex debug 'DEBUG=1' '')" "${MYEMAKEARGS[@]}" \
-		"$([ -f makefile.libretro ] && echo '-f makefile.libretro')" \
-		"$([ -f Makefile.libretro ] && echo '-f Makefile.libretro')"
+		"$([ -f makefile.libretro ] && echo '--file=makefile.libretro')" \
+		"$([ -f Makefile.libretro ] && echo '--file=Makefile.libretro')"
 }
```

Comment:
This fix is a nice workaround.

Overall this is a strange way to achieve the output to pass an argument to `emake`. Would rather do something like:

```shell
local makefile_args
makefile_args=()
if [ -f makefile.libretro ]; then
  makefile_args+=( -f makefile.libretro )
fi
emake ... "${makefile_args[@]}"
```

Also are the `CC=` and `CXX=` arguments really necessary? The default should already be set to the output of `tc-getCC`, etc.

I will however accept the workaround.